app/main.py

```diff
@@ -2,0 +1,11 @@
+def copy_file(command: str) -> None:
+    data = command.split()
+    file_to_copy = data[1]
+    copy = data[2]
```

Comment:
The variable names `file_to_copy` and `copy` could be more descriptive. Consider using names like `source_file` and `destination_file` to improve code readability and maintainability.